docs/Sources/tuist/tuist.docc/Articles/Tuist Cloud/Users/selective-testing.md

```diff
@@ -1,11 +1,34 @@
 # Selective Testing
 
-Learn 
+Run only tests that have changed since the last successful test run
 
 ## Overview
 
-<!--@START_MENU_TOKEN@-->Text<!--@END_MENU_TOKEN@-->
+As your project grows, so does the amount of your tests. For a long time, running all tests on every PR or push to `main` takes tens of seconds. But this solution does not scale to thousands of tests your team might have.
 
-### Section header
+On every test run on the CI, you probably build a project with cleaned derived data and re-run all the tests, regardless of the changes. `tuist test` helps you to drastically decrease the build time and then running the tests themselves.
 
-<!--@START_MENU_TOKEN@-->Text<!--@END_MENU_TOKEN@-->
+### Using cache binaries
+
+When you call `tuist test`, tuist generates a project focusing on test targets only. It is similar to you running `tuist generate MyTestA MyTestB ...`, and so you automatically get the benefits of the [binary caching](./binary-caching).
```

Comment:
I'd move this to the documentation about binary caching. We can include a section there where we mention which commands support binary-caching. We can mention that it's "tuist generate" and "tuist test".